packages/apputils-extension/src/themesplugins.ts

```diff
@@ -113,13 +117,29 @@ export const themesPlugin: JupyterFrontEndPlugin<IThemeManager> = {
       execute: () => manager.toggleThemeScrollbars()
     });
 
+    commands.addCommand(CommandIDs.changeFont, {
+      label: args =>
+        args['enabled'] ? `${args['font']}` : trans.__(`waiting for fonts`),
+      isEnabled: args => args['enabled'] as boolean,
+      isToggled: args => manager.getCSS(args['key'] as string) === args['font'],
+      execute: args =>
+        manager.setCSSOverride(args['key'] as string, args['font'] as string)
+    });
+
+    const incrLabel = trans.__('+ Size');
+    const decrLabel = trans.__('- Size');
```

Comment:
Just wondering if using `Increase size` is prefered over `+ Size`

Reply:
My sense is that

![image](https://user-images.githubusercontent.com/2263641/101660823-998fe280-3a15-11eb-8d16-03d1321f62a1.png)

is more concise and understandable at a quick glance than

![image](https://user-images.githubusercontent.com/2263641/101660897-aad8ef00-3a15-11eb-8dd4-a25ebe6f6b9e.png)

so I, at least, prefer it. We can ask everyone's opinion at the meeting